test_scripts/Resumption/InteriorVehicleData/009_InteriorVD_resumption_all_modules_without_moduleId_IGN_OFF.lua

```diff
@@ -0,0 +1,56 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: Successful resuming of interior vehicle data after IGN_OFF
+--  in case app was subscribed to all default modules (without moduleId)
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app with REMOTE_CONTROL hmi type is registered and activated
+-- 3. App is subscribed to all default modules via GetInteriorVehicleData(moduleType)
+--
+-- Sequence:
+-- 1. IGN_OFF and IGN_ON are performed
+-- 2. App starts registration with actual hashId after SDL restart
+-- SDL does:
+-- - a. send RC.GetInteriorVehicleData(subscribe=true, moduleType, default moduleId) to HMI during resumption data
+--    for each default module
+-- - b. respond RAI(SUCCESS) to mobile app
```

Comment:
please add 
b. SDL get a successful response for each GetInteriorVehicleData request with isSubscribed=true
c. SDL respond RAI(success=true, result code = SUCCESS) to mobile app
d. update hashId after successful resumption

Reply:
Done